apps/playground/src/app/components/forms/checkbox/checkbox.component.ts

```diff
@@ -64,7 +65,7 @@ export class CheckboxComponent implements OnInit {
 
   public ngOnInit(): void {
     this.reactiveFormGroup = this.#formBuilder.group({
-      reactiveCheckbox: new FormControl(undefined),
+      reactiveCheckbox: new FormControl(undefined, [Validators.requiredTrue]),
```

Comment:
Is this an intentional change? the `required` input should be enough to make this required; the consumer shouldn't have to pass the validator. Or is this just ensuring it still works if they try to pass their own validators?

Reply:
Yes, the latter.